demo/cmd/tcp-client/tcp-client.go

```diff
@@ -53,27 +58,25 @@ func main() {
 				log.Error().Err(writeErr).Msg("Write error")
 				continue
 			} else {
-				fmt.Printf("Wrote %d bytes, msg written: [%s]\n", bytesWritten, requestMsg)
+				log.Debug().Msgf("Wrote %d bytes, msg written: [%s]\n", bytesWritten, requestMsg)
```

Comment:
This should be info, because it logs what message is being sent so it can be compared with the echo response.